roles/authelia/tasks/main.yml

```
@@ -19,28 +19,26 @@
     msg: "Your kernel version {{ ansible_kernel }} Does not meet the minimum requirements for Authelia. Please see https://github.com/pimlie/ubuntu-mainline-kernel.sh for a kernel upgrade script"
   when: "{{ kernel_version is not version(kernel_options[kernel_major_minor], '>=', true) }}"
 
-- name: Make authelia directory.
-  file:
-    path: "{{ volumes_root }}/authelia"
-    state: directory
-
 - name: Create necessary folders, with appropriate permissions.
   file:
     path: "{{ volumes_root }}/authelia/{{ item }}"
```

```suggestion
    path: "{{ volumes_root }}/{{ item }}"
```

because your items are called `Authelia/config`

---

Yea, actually this can be changed.

good catch